py3status/modules/nvidia_temp.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# encoding: utf-8
+
```

Comment:
please, don't add a python shebang and make sure your encoding line is `# -*- coding: utf-8 -*-` exactly (QA)